lib/deliver/app_screenshot.rb

```diff
@@ -113,6 +117,11 @@ def self.calculate_screen_size(path)
           [1536, 2008],
           [1536, 2048]
         ],
+        ScreenSize::IOS_IPAD_PRO => [
+          [2732, 2048],
+          [2048, 2732]
+          # missing resolutions with status bar, not sure what they should be
```

Comment:
I don't think it should be possible to upload screenshots without the status bar